workspaces/util/src/__tests__/squashWhitespace-test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { squashWhitespace } from "../squashWhitespace";
+
+describe("squashWhitespace", () => {
+  it("removes whitespaces between words if it exists", () => {
+    expect(squashWhitespace("hello   world")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the end of the string", () => {
+    expect(squashWhitespace("hello world        ")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the beginginng of the string", () => {
+    expect(squashWhitespace("         hello world")).toBe("hello world");
+  });
+
+  it("nothing should be changed", () => {
```

Comment:
note: the string titles are sometimes used in test output, it's therefore helpful to have a sense of what is being tested from the title alone (without seeing the code)

so for example "removes whitespace at the end of the string" is very clear but "nothing should be changed" is not so clear without more context